L/libflame/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+name = "libflame"
+
+using BinaryBuilder, Pkg
+
+version = v"5.2.0"
+
+# Collection of sources required to build lapack
+sources = [
+    GitSource("https://github.com/flame/libflame",
+              "646ed9075cc45aeffb53632a3ec88defb423fefa"),
+    DirectorySource("bundled"),
+]
+
+# Bash recipe for building across all *64-bit* platforms
+# We try to maintain consistency with the blis Yggdrasil build scripts.
+
+script = raw"""
+cd $WORKSPACE/srcdir/libflame
+
+# We might need newer `config.guess`` and `config.sub` files
+update_configure_scripts
+
+windows_flags=""
+if [[ "${target}" == *-w64-mingw32* ]]; then
+    windows_flags+=" --enable-windows-build "
+    # disable time & clock functions on windows (mingw): sys/times.h is missing
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows-remove-time.patch
+fi
+
+# - Compile and build a LAPACK compatibility layer with --enable-lapack2flame
+# - If a static library is not needed, use --disable-static-build
+# - Enable a dynamic build with --enable-dynamic-build
+./configure \
+    --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --enable-multithreading=openmp \
+    --enable-lapack2flame \
+    --enable-dynamic-build \
+    --enable-cblas-interfaces \
+    --disable-autodetect-f77-ldflags --disable-autodetect-f77-name-mangling \
+    $windows_flags
+
+make -j${nproc}
+make install
+
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc="musl"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("x86_64", "freebsd"),
+    Platform("x86_64", "windows"),
+    # Platform("x86_64", "macos"),
+    # Platform("aarch64", "macos"),
+]
+
+# Enable the following line if trying to link against Fortran code:
+# platforms = expand_gfortran_versions(platforms)
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    RuntimeDependency(PackageSpec(name="blis_jll", uuid="6136c539-28a5-5bf0-87cc-b183200dce32")),
```

Comment:
Does this actually require BLIS as the BLAS layer, or can this also work on our libblastrampoline system? It would be nicer if we can get it to use libblastrampoline (which then could use BLIS if the user selects it).

Reply:
Will give this a go.